core/templates/pages/exploration-player-page/learner-experience/conversation-skin-embed.directive.html

```diff
@@ -26,6 +26,7 @@
            ng-class="{ 'conversation-skin-animate-tutor-card-on-narrow': !canWindowShowTwoCards() && isCurrentSupplementalCardNonempty()}"
            style="margin-bottom: 100px">
         <tutor-card displayed-card="displayedCard"
+                    is-avatar-image-shown = "true"
```

Comment:
Follow existing style with regards to spacing around "=".

Also, this should be avatar-image-is-shown, and you would do `isAvatarImageShown: &avatarImageIsShown` or similar.

Reply:
Done.